kratos/geometries/quadrilateral_3d_4.h

```diff
@@ -408,65 +407,73 @@ template<class TPointType> class Quadrilateral3D4
      */
     double Area() const override
     {
-        // Old
-
-        /*Vector d = this->Points()[2] - this->Points()[0];
-        return( sqrt( d[0]*d[0] + d[1]*d[1] + d[2]*d[2] ) );*/
+        // Finite element way
+        Vector temp;
+        DeterminantOfJacobian( temp, GeometryData::GI_GAUSS_3 );

```

Comment:
I'm not sure if creating the Vector is not harmful for efficiency. I think you can move the jacobian implemenation to a private method passing an array_1d with correct size and then use it in both methods.

Reply:
Yeah, that can work